data/mods/slrb/moves.js

```diff
@@ -179,6 +179,44 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Fairy",
 	},
+	// Aus Elise
+	"Zwischenzung": {
+		accuracy: 100,
+		basePower: 120,
+		category: "Special",
+		desc: "Deals damage two turns after this move is used. At the end of that turn, the damage is calculated at that time and dealt to the opponent.",
+		shortDesc: "Hits two turns after being used.",
+		id: 'zwischenzung',
+		name: "Zwischenzung".
+		pp: 5,
+		priority: 0,
+		flags: {},
+		isFutureMove: true,
+		onTry(source, target) {
+			if (!target.side.addSlotCondition(target, 'futuremove')) return false;
+			Object.assign(target.side.slotConditions[target.position]['futuremove'], {
+			move: 'zwischenzung',
+			source: source,
+			moveData: {
+				id: 'zwischenzung',
+				name: "Zwischenzung",
+				accuracy: 100,
+				basePower: 120, 
+				category: "Special",
+				priority: 0,
+				flags: {},
+				effectType: 'Move',
+				isFutureMove: true,
+				type: 'Fairy',
+				},
+			});
+			this.add('-start', source, 'Doom Desire');
```

Comment:
So, regarding this line. I am not sure what you intended, because it’ll
show “x chose doom desire as it’s destiny!” If you want to have its own unique message I’ll give you suggestion to add.

Reply:
Yeah, I wasn't sure what I should/how I could set something to replace that. How would I directly replace the "chose doom desire as its destiny" text? 

As far as what I'd replace it with, I .mail'd Aus Elise but they've yet to respond (Monita says last online 20 days ago). ZwischenZUG means an "in between move", as in an unexpected ploy in chess, but zwischenzung doesn't mean anything as far as I can tell so I can't really guess through context.

Reply:
```suggestion
			this.add('-start', source, 'Future Sight');
```
Im just gonna say that Future Sight's game text would be good enough